src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -507,18 +510,22 @@ private List<DRSAccessMethod> getDrsAccessMethodsOnGcp(
         new DRSAccessMethod()
             .type(DRSAccessMethod.TypeEnum.HTTPS)
             .accessUrl(httpsAccessURL)
-            .region(region);
+            .region(region)
+            .authorizations(authorizationsBearerOnly);
 
     return List.of(gsAccessMethod, httpsAccessMethod);
   }
 
-  private List<DRSAccessMethod> getDrsSignedURLAccessMethods(String prefix, String region) {
+  private List<DRSAccessMethod> getDrsSignedURLAccessMethods(
+      String prefix, String region, boolean passportAuth) {
     String accessId = prefix + region;
+    DRSAuthorizations authorizationsBearerOnly = buildDRSAuth(passportAuth);
```

Comment:
Naming nit: might this make more sense as `authorizations` since it may include a passport?